File: NodeBackend/Models/BookingTime.js
module.exports = class BookingTime  {

    constructor(date, startTime, endTime, bookingTimeBookingId) {
        this.date = date; 
        this.startTime = startTime; 
        this.endTime = endTime; 
        this.bookingTimeBookingId = bookingTimeBookingId;
    
    }
}

File: ASP.NET-Backend/ASP.NET-Backend/Models/BookingTime.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ASP.NET_Backend.Models
{
	public class BookingTime
	{
		public int bookingTimeId { get; set; }
		public string date { get; set; }
		public string startTime { get; set; }
		public string endTime { get; set; }
		public int bookingTimeBookingId { get; set; }
	}
}
